scripts/linters/pylint_extensions.py

```diff
@@ -1772,45 +1772,37 @@ class BlankLineBelowFileOverviewChecker(checkers.BaseChecker):
         )
     }
 
-    def process_module(self, node):
-        """Process a module to ensure that there is a blank line below
+    def visit_module(self, node):
+        """Visit a module to ensure that there is a blank line below
         file overview docstring.
 
         Args:
             node: astroid.scoped_nodes.Function. Node to access module content.
         """
-
-        multi_line_indicator = b'"""'
-        file_content = read_from_node(node)
-        file_length = len(file_content)
-        triple_quote_counter = 0
-        empty_line_counter = 0
-        for line_num in python_utils.RANGE(file_length):
-            line = file_content[line_num].strip()
-            # Single line comment, ignore it.
-            if line.startswith(b'#'):
-                continue
-            triple_quote_counter += line.count(multi_line_indicator)
-
-            if line.endswith(b'"""') and triple_quote_counter == 2:
-                closing_line_index_of_fileoverview = line_num
+        # Check if the given node has docstring.
+        if node.doc is None:
+            return
+        line_number = node.fromlineno
+        # Iterate till the start of docstring.
+        while True:
+            line = linecache.getline(node.root().file, line_number).strip()
+            if line.startswith((b'"""', b'\'\'\'', b'\'', b'"')):
                 break
+            else:
+                line_number += 1
 
-        if triple_quote_counter == 2:
-            empty_line_check_index = closing_line_index_of_fileoverview
-            if empty_line_check_index < file_length - 1:
-                while file_content[empty_line_check_index + 1] == b'\n':
-                    empty_line_counter += 1
-                    empty_line_check_index += 1
-
-            if empty_line_counter > 1:
-                self.add_message(
-                    'only-a-single-empty-line-should-be-provided',
-                    line=closing_line_index_of_fileoverview + 1)
-            elif empty_line_counter == 0:
-                self.add_message(
-                    'no-empty-line-provided-below-fileoverview',
-                    line=closing_line_index_of_fileoverview + 1)
+        doc_length = len(node.doc.split(b'\n'))
+        line_number += doc_length
+        first_line_after_doc = linecache.getline(
```

Comment:
This looks good, see if we can find better way to find the line number of completion of docstring!

Reply:
No we can't as this is not a node.